### PR TITLE
feature: add end point for getting the log.json from a job. #72

### PR DESCRIPTION
**Pull request by basalt79**

### CODE REVIEW

File: src/main/java/com/saucelabs/saucerest/SauceREST.java [hunk 1]
```diff
@@ -849,10 +890,6 @@ public String uploadFile(File file, String fileName) throws IOException {
     }
 
     /**
-     * @deprecated use {@link #uploadFile(File, String, boolean)}.
-     *
-     * Uploads a file to Sauce storage.
-     *
```
Comment (sceiler) [inline src/main/java/com/saucelabs/saucerest/SauceREST.java:855]: I think we should leave this comment in until the version after next so current user will have enough time to change their code.

File: src/main/java/com/saucelabs/saucerest/SauceREST.java [hunk 2]
```diff
@@ -882,9 +919,6 @@ public String uploadFile(File file, String fileName, boolean overwrite) throws I
     }
 
     /**
-     * @deprecated use {@link #uploadFile(InputStream, String, boolean)}.
-     * Uploads a file to Sauce storage.
-     *
```
Comment (sceiler) [inline src/main/java/com/saucelabs/saucerest/SauceREST.java:887]: I think we should leave this comment in until the version after next so current user will have enough time to change their code.

Comment (basalt79) [reply src/main/java/com/saucelabs/saucerest/SauceREST.java]: done with https://github.com/saucelabs/saucerest-java/pull/105/commits/0f583de4576c4b8d329e23849ee67cee2d4ce03a

File: src/main/java/com/saucelabs/saucerest/SauceREST.java (hunk 1 above)
Comment (basalt79) [reply src/main/java/com/saucelabs/saucerest/SauceREST.java]: done with https://github.com/saucelabs/saucerest-java/pull/105/commits/0f583de4576c4b8d329e23849ee67cee2d4ce03a

**Comment by sceiler** [comment]
@basalt79 Hi, sorry for the delay. Your PR looks good to me. Thank you for it! Do you mind resolving the merge conflicts? Sorry for the inconvenience I know they are caused by merging newer PRs first.

**Comment by basalt79** [comment]
@sceiler done, but not sure why travis is not triggered

**Comment by sceiler** [comment]
@basalt79 I don't why it doesnt restart but can you just close and reopen the PR? I still see that there are conflicts reported for whatever reason.

**Comment by basalt79** [comment]
fired a new one https://github.com/saucelabs/saucerest-java/pull/126

**Comment by basalt79** [comment]
@sceiler maybe you can tag this also with `hacktoberfest-accepted`

**Comment by sceiler** [comment]
@basalt79 this shouldn't be needed because the whole repo is set to hacktoberfester as a topic. But I will do it anyway just to make sure. Let me know if this does not count towards your profile.

**Pull request closed without merging**